.eslintrc

```diff
@@ -21,6 +21,7 @@
         "location"
       ]}],
     "react/display-name" : [0],
+    "react/no-find-dom-node": [0],
```

Comment:
It looks like we use `findDOMNode` in only two places. Do you think it's worth the effort to fix this now?